### PR TITLE
en: add tidbmonitor dynamic configuration

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1414
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by ran-huang** [comment]
@mikechengwei PTAL, thanks!

File: en/enable-dynamic-configuration-for-tidbmonitor.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  imagePullPolicy: IfNotPresent
+```
+
+After you modify the `prometheusReloader` configuration, TidbMonitor restarts automatically. After the restart, the dynamic configuration feature is enabled. All configuration change related to Prometheus is dynamically updated.
+
+For more examples, refer to [monitor-dynamic-configmap](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-dynamic-configmap).
+
+## Disable the dynamic configuration feature
+
+To disable the dynamic configuration feature, remove `prometheusReloader` from the `spec` field of TidbMonitor.
```
Comment (qiancai) [inline en/enable-dynamic-configuration-for-tidbmonitor.md:42]:
```suggestion
To disable the dynamic configuration feature, remove the `prometheusReloader` configuration from the `spec` field of TidbMonitor.
```

**Review by qiancai — COMMENTED** [review]
@ran-huang  PTAL. Thanks.

**Comment by ran-huang** [comment]
@qiancai All suggestions accepted. PTAL again, thanks!

**Review by qiancai — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**